linux_files/setup

```diff
@@ -325,19 +338,16 @@ function fzfinstall {
 read -r -p "Would you like to download and install optional command line finder fzf? [y/N] " response
 if [[ "$response" =~ ^[yY]$ ]]
 then
-    CURDIR=$(pwd)
-    TMPDIR=$(mktemp -d)
-    cd $TMPDIR
+    createtmp
     git clone --depth 1 https://github.com/junegunn/fzf.git
     cd fzf
     ./install
-    cd $CURDIR
-    rm -r $TMPDIR
+    cleantmp
 fi
 }
 
 function explorerintegration {
-read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response
+read -r -p "Would you like to enable Windows Explorer shell integration? [y/N] " response       # this says it defaults to N, which I think is correct, but hitting [enter] still attempts install
```

Comment:
on line 360, `Install.reg` is accidentally copied as `Uninstall.reg`, I think this is the reason....

Reply:
Ah, I see. I will address with some user feedback.